packages/viz/examples/styles/helpers/colorBinsStyle.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" type="text/css" href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css">
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <section id="map"></section>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.js"></script>
```

Comment:
Regarding to mapbox-gl, I have reviewed deck.gl repo and they are not very consistent with versions, but in latests examples they use `https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.0/mapbox-gl.js`

As a rule of thumb, I guess we could always use the latest stable mapbox, unless deck.gl becomes more explicit. Currently, latests stable = 1.8.1

Reply:
I'm gonna update the examples. That version was there since the very beginning, so I'm gonna update them, and point our `@deck.gl/geo-layers` dependency to 8.1.0 to finally get the polygon clipping.